### ISSUE TITLE
Svg element(s) does not have a mechanism that allows an accessible name value to be calculated.

### ISSUE THREAD

**Issue by Kivylius** [airbnb/react-dates#1503]
**react-dates version**
react-dates@18.3.1

**Describe the bug**
Accessibility concern on SVG icons. The svg icons that are used in buttons should have an `alt` or `aria-label` tag(s) to be AA compliment just like how the buttons have them.

See: https://amp.levelaccess.net/public/standards/view_best_practice.php?violation_id=362

**Source code (including props configuration)**
Any date picker that has SVG, such as close button or arrow button

e.g: `DateRangePickerInput.jsx`
```
 const closeIcon = customCloseIcon || (
    <CloseButton
      {...css(
        styles.DateRangePickerInput_clearDates_svg,
        small && styles.DateRangePickerInput_clearDates_svg__small,
      )}
      alt="close button"
    />
  );
```

**Screenshots/Gifs**
-

**Desktop (please complete the following information):**
 - OS: Any
 - Browser Chrome, IE, Firefox

**Is the issue reproducible in Storybook?**
Please link to the relevant storybook example

**Additional context**
The fixes are quite easy. alt tags should be provided for svg as they are images. In the example page the SVG are `aria-labeldby` an id, and that could be the button in react-dates example.

```
<!--SVG example with title element and aria-labelledby referencing the title element -->
<svg role="img" aria-labelledby="svg_title"
aria-description="svg_desc" width="400" height="400">
  <title id="svg_title">Purple Rectangle</title>
  <desc id="svg_desc">This is the description of the purple
rectangle</desc>
   <rect width="50" height="50" fill="purple"
aria-hidden="true">
</svg>
```
